app/assets/javascripts/host_checkbox.js.erb

```diff
@@ -42,10 +42,10 @@ function readFromCookie() {
 function toggle_actions() {
   var dropdown = $("#submit_multiple a");
   if ($.foremanSelectedHosts.length == 0) {
-    dropdown.addClass("disabled hide");
+    dropdown.addClass("disabled");
```

Comment:
does it make sense to have a pop over / hover on the btn when its disabled? - telling the user that he needs to select at least one host?